lib/src/read-contents/read-buffer.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 var fs = require('graceful-fs');
-var removeBomBuffer = require('remove-bom-buffer');
+var isUtf8 = require('is-utf8');
```

Comment:
Did we have a way around using this in removeBomStream?

Reply:
`removeBomStream` was changed to require encoding by `removeBomStream#8`. 
Before the change, `removeBomStream` was using `removeBomBuffer`.